udp_bridge/aes_helper.py

```diff
@@ -35,12 +35,12 @@ def encrypt(self, message: str) -> bytes:
         if self.encryption_key is None:
             return bytes(message, encoding="UTF-8")
 
-        return Fernet(key=self.encryption_key).encrypt(bytes(message, encoding="UTF-8"))
+        return Fernet(key=self.encryption_key).encrypt(message)
 
     def decrypt(self, enc: bytes) -> str:
         if len(enc) == 0:
             raise ValueError("Cannot decrypt empty data")
         if self.encryption_key is None:
             return str(enc, encoding="UTF-8")
 
-        return str(Fernet(key=self.encryption_key).decrypt(enc), encoding="UTF-8")
```

Comment:
As far as I'm aware the official documentation says that bytes are used as input to encrypt and output of decrypt.

https://cryptography.io/en/latest/fernet/#cryptography.fernet.Fernet.encrypt

Reply:
Sorry, I forgot to update the type hints. `message` is now bytes.